### PR TITLE
Fix conformance/rendering/blending for WebGL 1.0

### PR DESCRIPTION
**Pull request by lexaknyazev**

### CODE REVIEW

File: sdk/tests/conformance/rendering/blending.html [hunk 1]
```diff
@@ -146,7 +146,8 @@
         debug('Blending for RGBA8:');
 
         const gl = CreateContext();
-        fb = CreateValidFb(gl, [[gl.RGBA8, gl.RGBA, gl.UNSIGNED_BYTE]]);
+        // Do not use RGBA8 enum on WebGL 1.0
+        fb = CreateValidFb(gl, [[gl.texImage3D ? gl.RGBA8 : undefined, gl.RGBA, gl.UNSIGNED_BYTE]]);
```
Comment (kdashg) [inline sdk/tests/conformance/rendering/blending.html:150]:
```suggestion
        fb = CreateValidFb(gl, [[gl.texImage3D ? gl.RGBA8 : gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE]]);
```

Comment (kdashg) [reply sdk/tests/conformance/rendering/blending.html:150]: `undefined` works but I think it's nice to just ask for the right thing up-front.

**Review by kdashg — APPROVED** [review]

File: sdk/tests/conformance/rendering/blending.html (hunk 1 above)
Comment (lexaknyazev) [reply sdk/tests/conformance/rendering/blending.html:150]: Should similar updates be done to `RGBA16F` and `RGBA32F` below? Those rely on `undefined` being a special case in `CreateValidFb`.

**Review by kenrussell — APPROVED** [review]
Looks good to me, either way the internalformat is passed to CreateValidFb.

**Pull request merged**